src/bnet_stats_scraper.py

```diff
@@ -28,8 +29,27 @@ def get_stats(player, gateway):
     if gateway not in GatewayList:
       raise ValueError('Invalid Gateway')
 
-    page = urlopen('http://classic.battle.net/war3/ladder/' + GameInfo.get_game_version_str() + '-player-profile.aspx?Gateway=' + gateway + '&PlayerName=' + player.name)
-    soup = BeautifulSoup(page, 'html.parser')
-    BNetStatsScraper.__get_stat(soup, player.solo_stats, 'Solo Games')
-    BNetStatsScraper.__get_stat(soup, player.team_stats, 'Team Games')
-    BNetStatsScraper.__get_stat(soup, player.ffa_stats, 'FFA Games')
+    try:
+      # try grabbing stats from battle.net
+      url = 'http://classic.battle.net/war3/ladder/' + GameInfo.get_game_version_str() + '-player-profile.aspx?Gateway=' + gateway + '&PlayerName=' + player.name
+      page = urlopen(url, timeout=10)
```

Comment:
Do you feel like 10 is still a good value with the fallback?

Reply:
I was really unsure about the timouts. In my experience battle.net often takes some time to respond, so I thought it might be good to prio battle.net scraping. another option could be to run both calls async and decide which result to return to the player, whats your take on this?

Reply:
Lets not overcomplicate things. Don't worry about parallel scraping and just keep the 10s